### ISSUE TITLE
TimelineItem accessibilityInformation#role not working

### ISSUE THREAD

**Issue by JacksonKearl** [microsoft/vscode#101442]
Testing #100936

I don't see anything related to roles in the voiceover, and inspecting the DOM it seems like they're all `treeitem`s

![image](https://user-images.githubusercontent.com/8586769/86162954-70614280-bac4-11ea-96f3-d67b936df90d.png)

![image](https://user-images.githubusercontent.com/8586769/86162870-4f005680-bac4-11ea-869e-32a7cf5c994c.png)

**Comment by JacksonKearl** [comment]
Similar for status bar: 
```ts
const sbi = vscode.window.createStatusBarItem(
	{
		id: 'dfsddsf', name: 'hello',
		accessibilityInformation: { label: 'I am also a label', role: 'checkbox' }
	})
sbi.text = 'My sbi'
sbi.command = 'solunar-timeline.log'
sbi.show()
```
![image](https://user-images.githubusercontent.com/8586769/86164318-6b9d8e00-bac6-11ea-852b-c88ed4c15d56.png)

**Comment by isidorn** [comment]
@JacksonKearl great catch. Since we were not changing the role of the status bar in practice we forgot to pass it through. Similar for the TimelineItem.
I have fixed this such that now the `role` should be respected when passed. Thanks!
